documentation/modules/oauth/con-mapping-keycloak-authz-services-to-kafka-model.adoc

```diff
@@ -0,0 +1,92 @@
+[id="con-mapping-keycloak-authz-services-to-kafka-model_{context}"]
+= Mapping Keycloak Authorization Services to the Kafka authorization model
+
+[role="_abstract"]
+
+To grant Kafka permissions to user accounts or service accounts, use Keycloak Authorization Services rules on the OAuth client that represents the Kafka Broker.
+Typically, the client id of the broker OAuth client is `kafka`.
+
+Your OAuth 2.0 client definition for the Kafka Broker must have the *Authorization Enabled* option enabled.
+
+All permissions exist within the scope of this OAuth 2.0 client. If you have different Kafka clusters configured with different OAuth 2.0 client IDs, they each need a separate set of permissions even though they're part of the same Keycloak realm.
```

Comment:
```suggestion
All permissions exist within the scope of the `kafka` client. If you have different Kafka clusters configured with different OAuth client IDs, they each need a separate set of permissions even though they're part of the same Keycloak realm.
```

Reply:
I'm tending to think we don't use Oauth 2.0 everywhere. And just use OAuth. As long as we mention upfront in the OAuth section that it's OAuth 2.0?

Reply:
Yeah, it's actually easier to read that way.